File: javascript/fizzbuzz-2.js
function isMultipleOf(number, div) {
    return number % div === 0;
}

for (let i = 0; i <= 100; i++) {
    let output = "";
    if (isMultipleOf(i, 3)) { output += "Fizz"; }
    if (isMultipleOf(i, 5)) { output += "Buzz"; }
    if (output === "") { output = i; }

    console.log(output);
}

File: python/fizzbuzz-2.py
def fizz(i) -> bool:
    return i % 3 == 0

def buzz(i) -> bool:
    return i % 5 == 0

for i in range(101):
    output = ""
    if fizz(i):
        output += "Fizz"
    if buzz(i):
        output += "Buzz"
    if not output:
        output = str(i)

    print(output)